Bottom: 3ec352561a8dad6e458ba05cbeb2504d510368d1
Top:    d9b582cc1e1bdbef59cc92c157df066add486c60
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 10:00:19 +0800

Refresh of autofs-5.0.7-fix-hesiod-check-error-and-use-correct-LIBS-setting.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 5751805..3c829cf 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -58,6 +58,7 @@
 - add after sssd dependency to unit file.
 - dont start readmap unless ready.
 - fix typo forced-shutdown should be force-shutdown.
+- fix hesiod check error and use correct $(LIBS) setting.
 
 25/07/2012 autofs-5.0.7
 =======================
